docs/tutorials/intro.md

```diff
@@ -228,7 +290,7 @@ git checkout icu@1.0.0
 cargo run --bin icu4x-datagen --features bin -- \
     --cldr-tag latest \
     --icuexport-tag latest \
-    --out ~/projects/icu/icu4x-data \
+    --out ~/projects/icu_tutorial/icu4x-data \
```

Comment:
```suggestion
    --out data \
```

Reply:
Nope

Reply:
Not from a technical POV, I mean from a tutorial POV, i think this absolute path is fine since it's not in the `myapp` folder

It's a reminder that we're stepping out of that folder.